### ISSUE TITLE
Examples of learning dynamics

### ISSUE THREAD

**Issue by tcstewar** [ctn-waterloo/modelling_ideas#104]
I've seen a number of talks of online learning of dynamics (i.e. observe a lorenz attractor, do online learning of weights, and result in a system that produces the lorenz dynamics).  For example:

https://arxiv.org/abs/1902.02875

I know @arvoelke and @celiasmith have implemented this in nengo, (I believe based on https://github.com/arvoelke/nengolib/blob/master/docs/notebooks/examples/full_force_learning.ipynb ) but I don't think we've done a broad set of examples showing this.

**Comment by arvoelke** [comment]
Agreed! This would also make an excellent opportunity for a review paper.

**Comment by tcstewar** [comment]
It's also worth noting that the elife paper used nengo to implement the model, and the code is available at https://github.com/adityagilra/FOLLOW
